draft-ietf-core-object-security.md

```diff
@@ -691,11 +677,11 @@ The "application/oscon" media type is added to the Media Types registry:
 
         Encoding considerations: binary
 
-        Security considerations: See Appendix C of [[this document]].
+        Security considerations: See Appendix C of this document.
```

Comment:
You generally want to do an internal cross-reference instead of naming appendices by name in the source.